### DISCUSSION TITLE
Hitlight behave differently between stable in kiai in osu ruleset

### DISCUSSION THREAD

**Discussion by cdwcgt** [ppy/osu#24725] [Q&A]
in stable, hitlight will flash when a new beat in kiai,

image from stable:
![image](https://github.com/ppy/osu/assets/34775378/af324aca-0abf-4218-8f54-ba99fb28dfdd)

hit circle 1 is not hit but hit lighting flash because this is in kiai time.

image from lazer and use the same skin:
![image](https://github.com/ppy/osu/assets/34775378/b3d76d47-1300-4ec3-b052-e2e03c9d2c37)

This is more important for playing HD (especially with FL)

But from the feeling, he has a feeling of cheating? not sure

**Comment by peppy** [comment]
probably intende

**Reply by cdwcgt** [reply]
Is it possible to implement it on lazer, or will this feature be dropped?
